Standards/scs-XXXX-vN-mandatory-and-supported-IaaS-services.md

```diff
@@ -0,0 +1,77 @@
+To fulfill all SCS-provided standards only a subset of these APIs are required.
+Some more but not all remaining OpenStack APIs are also supported additionally by the SCS project and may be part of its reference implementation.
+This results in different levels of support for specific services.
+This document will give readers insight about how the SCS classifies the OpenStack services accordingly.
+If a cloud provides all mandatory and any number of supported OpenStack APIs, it can be tested for SCS-compliance.
+Any unsupported services will not be tested.
+
+## Mandatory IaaS APIs
+
+The following IaaS APIs MUST be present in SCS-compliant IaaS deployments and could be implemented with the corresponding OpenStack services:
+
+| Mandatory API | corresponding OpenStack Service | description |
+|-----|-----|-----|
+| **block-storage** | Cinder | Block Storage service |
+| **image** | Glance | Image service |
+| **identity** | Keystone | Identity service |
+| **network** | Neutron | Networking service |
+| **compute** | Nova | Compute service |
+| **load-balancer** | Octavia | Load-balancer service |
+| **s3** or **object-store** | S3 API object storage | No formal standard exists, many implementations: Swift, RadosGW, minio... |
+
+:::caution
+
+S3 API implementations may differ in certain offered features.
+CSPs must publicly describe, which implementation they use in their deployment.
+Users should always research whether a needed feature is supported in the offered implementation.
+
+:::
+
+## Supported IaaS APIs
+
+The following IaaS APIs MAY be present in SCS-compliant IaaS deployment, e.g. implemented thorugh the corresponding OpenStack services, and are considered in the SCS standards.
+
+| Mandatory API | corresponding OpenStack Service | description |
+|-----|-----|-----|
+| **key-manager** | Barbican | Key Manager service |
+| **billing** | Cloudkitty | Rating/Billing service |
+| **telemetry** | Ceilomete | Telemetry service |
+| **dns** | Designate | DNS service |
+| **time-series-databse** | Gnocchi | Time Series Database service |
+| **orchestration** | Heat | Orchestration service |
+| **bare-metal** | Ironic | Bare Metal provisioning service |
+| **shared-file-systems** | Manila | Shared File Systems service |
+| **ha** | Masakari | Instances High Availability service |
+
+## Unsupported IaaS APIs
+
+All other OpenStack services, which APIs are not mentioned in the mandatory or supported lists will not be tested for their compatibility and conformance in SCS clouds by the SCS community.
+Those services MAY be integrated into IaaS deployments by a Cloud Service Provider on their own responsibility but the SCS will not assume they are present and potential issues that occur during deployment or usage have to be handled by the CSP on their own accord.
+The SCS standard offers no guarantees for compatibility or reliability of services categorized as unsupported in conjunction with an SCS-conformant infrastructure.
```

Comment:
just personal opinion, I would shorten that sentence to make it easier to understand/read:

```suggestion
The SCS standard offers no guarantees for compatibility or reliability of services categorized as unsupported.
```